site_scons/env_modules.py

```diff
@@ -188,3 +190,15 @@ def load_mpi(mpi):
     if _env_module.env_module_init is None:
         _env_module.env_module_init = _env_module()
     return _env_module.env_module_init.load_mpi(mpi)
+
+def show_avail():
+    """ global function to show the available modules"""
+    if _env_module.env_module_init is None:
+        _env_module.env_module_init = _env_module()
+    return _env_module.env_module_init.show_avail()
+
+def get_module_list(key):
+    """ global function to show the modules that map to a key"""
+    if _env_module.env_module_init is None:
+        _env_module.env_module_init = _env_module()
+    return _env_module._mpi_map[key]
```

Comment:
(pylint-protected-access) Access to a protected member _mpi_map of a client class